### DISCUSSION TITLE
View changes to source in editor

### DISCUSSION THREAD

**Discussion by NextPage66** [codefori/vscode-ibmi#2180] [General]
How can I see only the source lines that have changes since the last save?

**Comment by sebjulliand** [comment]
You need to use Git to do this. There is no other way, as far as I know.

**Comment by worksofliam** [comment]
You may want to try 'Compare Active File with Saved' in the command palette.

<img width="644" alt="image" src="https://github.com/user-attachments/assets/2afdbfe2-7580-44fb-affb-d453ad573c9e">

**Reply by NextPage66** [reply]
Great idea....beautiful.  I was looking in the menus for this but could not find it.  Thanks

**Reply by SJLennon** [reply]
@worksofliam 
The above works, but...

I have a member open in the editor. If I right click on the member in the object browser, what does "Compare with active file " do? 
![image](https://github.com/user-attachments/assets/d8147920-847a-4dc2-ac71-fd928e564576)

 It says:
![image](https://github.com/user-attachments/assets/a9670c8f-9dbf-4a21-8a78-90bf416df9d5)

I thought it might compare the member in the object browser to the open file in the editor so I could see what I had changed since the last save, but it apparently does nothing, and there is nothing in the Code for IBM i output.
